config.json

```diff
@@ -3,6 +3,6 @@
     "config"         : "",
     "asap_scheduler" : false,
     "lp_scheduler"   : false,
-    "out"            : ""
+    "sv_out"         : ""
```

Comment:
I don't think this is intuitive to understand. We might have other options which affect SystemVerilog output and this will affect the understanding of CLI.

Reply:
It is an assumption only, not a fact.
I have another idea. Why not to name the option just "--out" or "--format" and to get the output file's language from it's extension?

Reply:
Comment was discussed IRL was agreed to be closed.